index.php

```diff
@@ -4311,7 +4339,7 @@ function openEEOTable() {
               "<option value='"+msaCode+"' class='option'>"+msaName+"</option>"
```

Comment:
I want to change this back to ES6 syntax later (I changed it because notepad++ didn't like it when in a php file), but for now it's fine